ragna/deploy/_ui/styles.py

```diff
@@ -1,155 +1,86 @@
-                            background-color: white !important;
-                            height: var(--ragna-accordion-header-height);
-                            padding-top: 0px;
-                            padding-bottom: 0px;
-                            outline:0px;
-                            margin-left: 15px;
-                            margin-right: 15px;
-                            width: calc(100% - 30px);
-                            border-bottom: 2px solid #D9D9D9;
-                        
-                        }
-                        
-                        button.accordion-header div.card-button {
-                            font-size: 11px;
-                            padding-top: 5px;
-                            margin-left: 0px;
-                            margin-right: 10px;
-                        }
-                        
-                        div.card-header-row {
-                            height: var(--ragna-accordion-header-height);
-                            background-color: unset !important;
-                        }
 
-                        /* styles the content of the sources content (the expanding areas of the Accordion) */
-                        div.bk-panel-models-markup-HTML.markdown {
-                            margin-left: 15px;
-                            margin-right: 15px;
-                            margin-top:0px;
-                        }
+def apply_design_modifiers():
+    css_filepaths = []
+    for dir, classes in css_modifiers.items():
+        for cls in classes:
+            css_filename = cls.__name__.lower().split(".")[-1] + ".css"
+            add_modifier(cls, f"css/{dir}/{css_filename}")
+            css_filepaths.append(f"css/{dir}/{css_filename}")
 
-                    """
-        ]
-    }
+    return css_filepaths
 
-    pn.theme.fast.Fast.modifiers[pn.pane.HTML] = {
-        "stylesheets": [
-            """ :host(.card-title) {
-                                height: var(--ragna-accordion-header-height);
-                                margin: 0px;
-                            }
-                        
-                            :host(.card-title) div {
-                                height: var(--ragna-accordion-header-height);
-                                
-                                display:flex;
-                                flex-direction:row;
-                                align-items:center;
-                            }
-                        
 
-                            :host(.card-title) h3 {
-                                font-weight: normal;
-                            }
-                        """
-        ]
-    }
+def add_modifier(
+    modifier_class: pn.viewable.Viewable,
+    modifications: Dict[str, Any],
+    property: str = "stylesheets",
+):
+    if modifier_class not in pn.theme.fast.Fast.modifiers:
+        pn.theme.fast.Fast.modifiers[modifier_class] = {}
```

Comment:
Just a little syntax sugar

```suggestion
    properties = pn.theme.fast.Fast.modifiers.setdefault(modifier_class, {})
    property_modifications = properties.setdefault(property, [])
    property_modifications.append(modifications)
```

This also supersedes the `if` / `else` below, but GH doesn't let me suggest it for both.